terraform/slurm_cluster/modules/_instance_template/main.tf

```diff
@@ -103,7 +107,7 @@ resource "google_compute_instance_template" "tpl" {
   dynamic "service_account" {
```

Comment:
It shouldn't be dynamic, always create exactly one `service_account`.

Reply:
Done